pkg/rpc/server.go

```diff
@@ -322,21 +322,22 @@ func (s *Server) getAccountState(reqParams Params, unspents bool) (interface{},
 
 // invokescript implements the `invokescript` RPC call.
 func (s *Server) invokescript(reqParams Params) (interface{}, error) {
-	hexScript, err := reqParams.ValueWithType(0, stringT)
-	if err != nil {
-		return nil, err
+	if len(reqParams) < 1 {
+		return nil, errInvalidParams
 	}
-	script, err := hex.DecodeString(hexScript.GetString())
+
+	script, err := reqParams[0].GetBytesHex()
 	if err != nil {
-		return nil, err
+		return nil, errInvalidParams
 	}
+
 	vm, _ := s.chain.GetTestVM()
 	vm.LoadScript(script)
 	_ = vm.Run()
 	result := &wrappers.InvokeResult{
 		State:       vm.State(),
 		GasConsumed: "0.1",
-		Script:      hexScript.GetString(),
+		Script:      reqParams[0].GetString(),
```

Comment:
so we're taking just the first param only?

Reply:
Yep, all the other ones are irrelevant for the invoker.

Reply:
We could probably check for these excessive parameters and fail, but that's a different story.